### DISCUSSION TITLE
Insert/ retrieve relationsships

### DISCUSSION THREAD

**Discussion by svenhue** [CodeDredd/pinia-orm#7] [Q&A] [answered]
So i try to insert data with relationsship... but i cant retrieve the data of the relationsship

![image](https://user-images.githubusercontent.com/83905274/163580338-b905c340-3e40-46f4-9b25-743c632191be.png)

![image](https://user-images.githubusercontent.com/83905274/163582346-be2a20d7-c499-44b7-a0e9-99693bd85f52.png)

![image](https://user-images.githubusercontent.com/83905274/163580286-0c86f14c-1e13-4de6-ad50-d26b007fc6a4.png)

![image](https://user-images.githubusercontent.com/83905274/163582317-0271b0f1-c0aa-4652-8675-f829b1de25d6.png)

console.log(resullt) ....

![image](https://user-images.githubusercontent.com/83905274/163580462-c1b80acf-b25e-4307-9fef-7c0ede039d23.png)

**Comment by CodeDredd** [comment, marked as answer]
Had to take look at the `fresh` function

**Reply by svenhue** [reply]
with save method and then retrieving with "with("components)"it works
